app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -14,6 +15,16 @@ def update
         redirect_back(fallback_location: spree.admin_taxjar_settings_path)
       end
 
+      def sync_nexus_regions
+        if ENV["TAXJAR_API_KEY"]
+          @nexus_regions = Rails.cache.fetch(:nexus_regions, expires_in: 2.hours) do
+            SuperGood::SolidusTaxjar.api.nexus_regions
+          end
+          flash[:success] = "Updated with new Nexus Regions"
+        end
```

Comment:
I think we should return a bad status code or something in the `else` here! Performing a successful redirect after failing to accomplish the goal of the endpoint seems conflicting.

Reply:
We added a failure message in the `else`!